tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+
+* `[abstract]`
+* `[acknowledgments]`
+* `[appendix]`
+* `[bibliography]`
+* `[index]`
+* `[preface]`
+
+There are also some Metanorma-specific section identifiers:
+
+* `[terms and definitions]`
+* *Any more?* 
+
+[source, AsciiDoc]
+----
+= Document title
+
+== Abstract
+
+== Foreword
+
+[preface] <1>
+== Introduction to version 3 of this standard
+
+[bibliography] <2>
+== Normative references
+
+[heading=terms and definitions] <3>
+== Terms, definitions, and abbreviations
+
+
+[bibliography]
+== Bibliography
+	...
+[appendix,obligation=informative] <4>
+== Annex
+	...
+----
+
+<1> This section is an introduction but the compiler can not recognize it because it deviates from plain `== Introduction`. The `preface` attribute ensures that the document structure is still valid.
+<2> The `Normative references` section contains references to related standards. The compiler needs the `[bibliography]` tag, so that it renders the reference correctly.
+<3> An alternative way to assign a section identifier is use the `heading` tag. 
+<4> Here, the appendix is called “Annex”, and needs the `appendix` tag to define what kind of section it is. You can specify if a section is normative or informative by adding the `obligation` identifier after the section identifier. 
+
+[[exercise-2-2]]
+include::exercises.adoc[tag=exercise-2-2]
+
+We’ve built the basic structure of a Metanorma document. Let’s add some content to our document in the next lesson.
+>> Start next lesson 
+
+=== Blocks
+
+Information that forms a logical segment, such as a paragraph or a list, is called a block.
+A block starts and ends with a https://asciidoctor.org/docs/asciidoc-writers-guide/#delimited-blocks[delimiter] which is a sequence of characters. The delimiters tell the compiler that the following input belongs together. 
+
+.Examples for different blocks
+[source, AsciiDoc]
+------
+I’m a paragraph and I don’t need a block delimiter. 
+<1>
+. I’m a list item and also do not need a delimiter
+. I’m the second list item
+. I'm list item number three
+
+//// <2>
+I’m a comment.
+////
+
+[source, python] <3>
+----
+print("Hello World!")
+----
+------
+
+<1> Paragraphs and lists are types of blocks that do not need any delimiters. They are separated by a blank line in between blocks.
+<2> Comments use four slashes as a delimiter `////`. No other block uses slashes to begin and end a block.
+<3> All other types of blocks rely on delimiters, such as four dashes `----` or four equal signs `====`. This block contains a code sample written in python.
+
+
+Apart from the blocks shown in the example, there are many types of blocks in AsciiDoc, such as:
+* Paragraphs
+* Lists
```

Comment:
You've mentioned these two and code samples. Remove, or remove the "Apart from".